packages/addons-website/scss/components/_swatch-palette-widget.scss

```diff
@@ -0,0 +1,89 @@
+@import '../prefix';
+
+.#{$prefix}--swatch-palettes {
+  display: flex;
+  flex-direction: column;
+  .#{$prefix}--swatch-palette {
+    float: left;
+    width: 100%;
+    margin-bottom: 2rem;
+    .#{$prefix}--swatch {
+      padding: 1rem;
+      font-family: IBM Plex Mono,Menlo,DejaVu Sans Mono,Bitstream Vera Sans Mono,Courier,monospace;
```

Comment:
I think we can use `@include carbon--type-style(code-02')` here instead of setting the font family